File: Phyton2/4.py
for son in range(100, 1000):
    a = son % 10
    b = (son // 10) % 10
    c = son // 100
    if ((a == b and a != c) or (a == c and a != b) or (b == c and b != a)) and not (a == b == c):
        print(son)

File: imtihon/4.c
#include <stdio.h>

void func(int *son) {

    if(*son >=10 && *son <= 99) {
        printf("%d%d", *son%10, *son/10);
    } else printf(" 2 xonali son kiriting!");
}

int main () {

    int a;
    printf(" Son kiriting: ");
    scanf("%d",&a);

    func(&a);

    return 0;
}